packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library"
+import classnames from "classnames"
+import {
+  FieldGroup,
+  FieldMessage,
+  Input,
+  InputProps,
+  Label,
+} from "@kaizen/draft-form"
+
+import styles from "./DateInput.scss"
+
+type OmittedInputProps =
+  | "startIconAdornment"
+  | "endIconAdornment"
+  | "inputType"
+  | "inputValue"
+
+export interface validationMessagesProps {
+  error?: string | React.ReactNode
+}
+
+export interface DateInputProps extends Omit<InputProps, OmittedInputProps> {
+  id: string
+  calendarId?: string
+  buttonRef?: React.RefObject<HTMLButtonElement>
+  labelText: string | React.ReactNode
+  inline?: boolean
```

Comment:
This inline prop can go away as well, and just hardcode to true where it was being used